src/components/app/app.tsx

```diff
@@ -0,0 +1,14 @@
+import MainPage from '../../pages/main/main.tsx';
+
+type AppTotalOffersProps = {
+  totalOffers: number;
+}
+
+
+function App({totalOffers}: AppTotalOffersProps): JSX.Element {
+  return (
+    <MainPage totalOffers = {totalOffers}/>
```

Comment:
пробелы в атрибутах в 99% случаев не используют
```suggestion
    <MainPage totalOffers={totalOffers}/>
```